rosco-manifests/src/main/java/com/netflix/spinnaker/rosco/manifests/helm/HelmTemplateUtils.java

```diff
@@ -96,4 +103,12 @@ private Path downloadArtifactToTmpFile(BakeManifestEnvironment env, Artifact art
     artifactDownloader.downloadArtifactToFile(artifact, targetPath);
     return targetPath;
   }
+
+  private String getHelmExecutableForRequest(HelmBakeManifestRequest request) {
+    if (BakeManifestRequest.TemplateRenderer.HELM2.equals(request.getTemplateRenderer())) {
+      return helmConfigurationProperties.getV2ExecutablePath();
+    } else {
+      return helmConfigurationProperties.getV3ExecutablePath();
+    }
```

Comment:
Small nit, no need for the `else` here since you're returning early in the `if`.

Reply:
Or maybe a `switch` here?  It's fine now, but this will start getting way more complex if we ever add any other condition.